### PR TITLE
Clean and fix the Movie Maker button

### PR DESCRIPTION
**Pull request by YeldhamDev**
- Color the pressed version of the icon red instead of using a new one.
- Increase slightly the non-pressed alpha color so it doesn't look disabled.
- Change logic of neighborhood icons so they don't stay pressed if run state is stopped by the Movie Maker.

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -6741,15 +6730,11 @@ EditorNode::EditorNode() {
...
-	// Restore these values to something more useful so it ignores the theme
-	write_movie_button->add_theme_color_override("icon_normal_color", Color(1, 1, 1, 0.4));
-	write_movie_button->add_theme_color_override("icon_pressed_color", Color(1, 1, 1, 1));
-	write_movie_button->add_theme_color_override("icon_hover_color", Color(1.2, 1.2, 1.2, 0.4));
-	write_movie_button->add_theme_color_override("icon_hover_pressed_color", Color(1.2, 1.2, 1.2, 1));
-	write_movie_button->add_theme_color_override("icon_focus_color", Color(1, 1, 1, 1));
-	write_movie_button->add_theme_color_override("icon_disabled_color", Color(1, 1, 1, 0.4));
+	// This button behaves differently, so color it as such.
+	write_movie_button->add_theme_color_override("icon_normal_color", Color(1, 1, 1, 0.7));
+	write_movie_button->add_theme_color_override("icon_pressed_color", Color(0.93, 0.33, 0.33));
```
Comment (Calinou) [inline editor/editor_node.cpp:6736]: I recommend reusing the editor error color here:

```suggestion
	write_movie_button->add_theme_color_override("icon_pressed_color", get_theme_color(SNAME("error_color"), SNAME("Editor")));
```

The name is not very semantic for this use case, but it should be more visible on light theme (as the color is automatically adjusted in that case).

Comment (YeldhamDev) [reply editor/editor_node.cpp]: Done.

**Comment by KoBeWi** [comment]
>Color the pressed version of the icon red instead of using a new one.
>Increase slightly the non-pressed alpha color so it doesn't look disabled.

You could use the same colors as record button in AnimationPlayer (if you aren't already)
godot windows tools 64_THOoP9NMbn

**Review by KoBeWi — APPROVED** [review]
Looks ok.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**